package.json

```
@@ -124,7 +124,7 @@
     "escape-string-regexp": "^4.0.0",
     "eslint-scope": "^8.4.0",
     "eslint-visitor-keys": "^4.2.1",
-    "espree": "^10.4.0",
+    "espree": "^11.0.0",
```

```suggestion
    "espree": "https://gitpkg.vercel.app/Pixel998/js/packages/espree?Program-range-cover-whole-program&scripts.postinstall=npm%20install%20--ignore-scripts%20%26%26%20npm%20run%20build",
```

We can use the branch of https://github.com/eslint/js/pull/677 to test changes. Before merging this PR (after espree v11 is released), we'll update this line.